src/Gridify/GridifyGlobalConfiguration.cs

```diff
@@ -46,6 +46,13 @@ public static class GridifyGlobalConfiguration
       /// Default is false
       /// </summary>
       public static bool DisableNullChecks { get; set; } = false;
+      
+      /// <summary>
+      /// By default, string comparison is case sensitive.
+      /// You can change this behavior by setting this property to true.
+      /// Default is false
+      /// </summary>
+      public static bool DefaultStringComparisonIsCaseInsensitive { get; set; } = false;
```

Comment:
Two improvements that I suggest here are:
- configuration name could be something less verbose, like `CaseInsensitiveSearch` or `CaseInsensitiveFiltering`
- We need to mirror the same configuration in the `GridifyMapperConfiguration` file, so people who are not using global configurations can also use it.

Reply:
Sounds good.